src/pages/resultsView/coExpression/CoExpressionVizUtils.ts

```diff
@@ -54,16 +55,23 @@ export function computePlotData(
     };
     for (const datum of mutationData) {
         if (datum.proteinChange) {
-            const targetData = (datum.entrezGeneId === xEntrezGeneId ? xMutations : yMutations);
+            const targetData = (datum.entrezGeneId === xGeneticEntityId ? xMutations : yMutations);
```

Comment:
wont this code break when genetic entities are genesets?

Reply:
@adamabeshouse No, because gene sets do not contain mutation data.

Reply:
ah okay. can you add a comment about that?